cmd/zfs/zfs_main.c

```diff
@@ -727,6 +727,38 @@ finish_progress(char *done)
 	pt_header = NULL;
 }
 
+/* This function checks if the passed fd refers to /dev/null or /dev/zero */
+#ifdef __linux__
+static int
+is_dev_files(int fd)
```

Comment:
the name of this is confusing and doesn't match what it does now

Reply:
Sure, done.